apps/opik-backend/src/main/java/com/comet/opik/infrastructure/EncryptionService.java

```diff
@@ -0,0 +1,59 @@
+package com.comet.opik.infrastructure;
+
+import com.google.inject.ImplementedBy;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import lombok.NonNull;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+import java.nio.charset.StandardCharsets;
+import java.security.Key;
+import java.util.Base64;
+
+@ImplementedBy(EncryptionServiceImpl.class)
+public interface EncryptionService {
+    String encrypt(String data);
+    String decrypt(String encryptedData);
+}
+
+@Singleton
+class EncryptionServiceImpl implements EncryptionService {
+
+    private static final String ALGO = "AES";
```

Comment:
Should we make it configurable?

Reply:
For now we agreed to use this encryption, same as for EM. And it's not reasonable to make it configurable. If we change it, we also would need to change implementation (for example key size or something else). It depends on algorithm.